src/@3rdweb-sdk/react/components/connect-wallet/index.tsx

```diff
@@ -64,8 +48,20 @@ export const CustomConnectWallet: React.FC<ConnectWalletProps> = ({
       privacyPolicyUrl="/privacy"
       hideTestnetFaucet={false}
       networkSelector={{
-        popularChains: favChains ?? popularChains,
-        recentChains,
+        sections: [
+          {
+            label: "Recently used",
+            chains: recentChains,
+          },
+          {
+            label: "Favorites",
+            chains: favChainsQuery.data ?? [],
+          },
```

Comment:
what if we only showed this if they had favorites? or does the section already behave that way that if its empty it hides?